packages/react-devtools-shared/src/__tests__/inspectedElementContext-test.js

```diff
@@ -530,7 +530,7 @@ describe('InspectedElementContext', () => {
       ['second', mapShallow],
     ]);
     const objectOfObjects = {
-      inner: {string: 'abc', number: 123, boolean: true},
+      inner: {str: 'ab', nb: 12, bool: true, [Symbol('sb')]: 'sb'},
```

Comment:
For easier review you could just add the additional symbol. Changing the other keys and values can be distracting.

Reply:
Done :)